### ISSUE TITLE
Accessibility: select dropdowns are not accessible

### ISSUE THREAD

**Issue by ethindp** [goauthentik/authentik#6825]
**Describe the bug**
Selection dropdowns (like those used in the stage management UI for changing the passwordless flow) are not accessible and appear as edit boxes to assistive technology.

**To Reproduce**

1. Launch a screen reader like NVDA
2. Go to edit a stage
3. Scroll down to any of the flow selection options and try to use the arrow keys when the dropdown is focused.

**Expected behavior**
I should be able to select any available authentication flow.

**Screenshots**
N/A

**Logs**
N/A

**Version and Deployment (please complete the following information):**

-   authentik version: 2023.8
-   Deployment: mash-playbook (docker)

**Additional context**
I am legally blind and use a screen reader.

**Comment by ethindp** [comment]
Any update? To solve this issue I have some recommendations:

1. Use normal HTML combo boxes. I know they aren't the most stylish, but they do work, and eye candy isn't everything IMO.
2. Use [ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA). (In general, no ARIA is better than using it, but clearly it needs to be used in this case.) For good, standardized recommendations on how ARIA should be used, check out the [ARIA authoring best practices guide](https://www.w3.org/WAI/ARIA/apg/practices) or the [ARIA patterns guide](https://www.w3.org/WAI/ARIA/apg/patterns).
